crates/networking/p2p/sync/trie_rebuild.rs

```diff
@@ -21,15 +21,20 @@ use tracing::{info, warn};
 use crate::sync::seconds_to_readable;
 
 use super::{
-    SyncError, MAX_CHANNEL_MESSAGES, MAX_CHANNEL_READS, MAX_PARALLEL_FETCHES,
-    SHOW_PROGRESS_INTERVAL_DURATION, STATE_TRIE_SEGMENTS_END, STATE_TRIE_SEGMENTS_START,
+    SyncError, MAX_CHANNEL_MESSAGES, MAX_CHANNEL_READS, SHOW_PROGRESS_INTERVAL_DURATION,
+    STATE_TRIE_SEGMENTS_END, STATE_TRIE_SEGMENTS_START,
 };
 
 /// The storage root used to indicate that the storage to be rebuilt is not complete
 /// This will tell the rebuilder to skip storage root validations for this trie
 /// The storage should be queued for rebuilding by the sender
 pub(crate) const REBUILDER_INCOMPLETE_STORAGE_ROOT: H256 = H256::zero();
 
+/// Max storages to rebuild in parallel
+const MAX_PARALLEL_REBUILDS: usize = 15;
+
+const MAX_SNAPSHOT_READS_WITHOUT_COMMIT: usize = 5;
```

Comment:
what's the criteria for this number?

Reply:
Trial and error mostly. The gist is that less commits make rebuild go faster but large commits can slow down other processes. I sought the highest value that would not slow down state sync.

Reply:
Wonder if these values that might be ideal for Holesky, are also appropriate for the other networks

Reply:
These values should be the least affected by a change of network as rebuild happens in the background and only interacts with the DB, it is not really affected by peer availability/speed at which we receive data from peers, and we cannot really split it into different processes to scale to a bigger trie size.
They could possibly change along with changes/improvements to trie caches and the DB.
These are also strong candidates to become customizable via config file (such as in #2331 )